### PR TITLE
OSDOCS-7366: Release note; CIDR removed MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14 only

Issue:
https://issues.redhat.com/browse/OSDOCS-7366

Link to docs preview:
https://63275--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-14-release-notes#microshift-4-14-notable-technical-changes
https://63273--docspreview.netlify.app/microshift/latest/microshift_configuring/microshift-using-config-tools#microshift-yaml-default_microshift-configuring (updated section; Netlify hasn't picked up the merge from related PR)

QE review:
- [x] QE has approved this change.

Additional information:
Doc change is here: https://github.com/openshift/openshift-docs/pull/63273

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@zshi-redhat PTAL and let me know if LGTM?

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 1]
```diff
@@ -115,6 +115,25 @@ $ ip route
...
+[id="microshift-4-14-networking-changes"]
+=== Networking changes
+
+Networking changes to {product-title} {product-version} include IP address assignment changes, gateway changes, and other refinements.
+
+[discrete]
+[id="microshift-4-14-cidr-removal"]
+==== CIDR removed
+
+CIDR notation for grouping IP addresses was removed with this release.
```
Comment (zshi-redhat) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:135]: Could you mention the `clusterNetwork` config where CIDR notation was removed? 
Just to make it clear on what user-facing config is updated.

Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-14-release-notes.adoc]: Yes, good note, thank you. Added detail and xref to the yaml example in the docs.

**Comment by ShaunaDiaz** [comment]
@zshi-redhat What do you think? Look good to you?

**Comment by ShaunaDiaz** [comment]
@dhensel-rh Will you PTAL?

**Comment by zshi-redhat** [comment]
> @zshi-redhat What do you think? Look good to you?

LGTM

**Comment by xenolinux** [comment]
Looks good to me!

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead.

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 2]
```diff
@@ -115,6 +115,24 @@ $ ip route
...
+== Notable technical changes
+
+{product-title} {product-version} introduces the following notable technical changes.
+// Note: use [discrete] for these sub-headings.
+
+[discrete]
+[id="microshift-4-14-networking-changes"]
+=== Networking changes
+
+Networking changes to {product-title} {product-version} include IP address assignment changes, gateway changes, and other refinements.
```
Comment (abhatt-rh) [inline microshift_release_notes/microshift-4-14-release-notes.adoc:128]: "..., and other refinements" reads a bit vague. Consider providing specific information.

**Review by abhatt-rh — COMMENTED** [review]
@ShaunaDiaz and @xenolinux: Peer review follow-up complete.
I have added a few suggestion for your consideration.

File: microshift_release_notes/microshift-4-14-release-notes.adoc [hunk 3]
```diff
@@ -115,6 +115,24 @@ $ ip route
...
+
+[discrete]
+[id="microshift-4-14-cidr-removal"]
+==== CIDR notation removed from configuration
+
+The CIDR notation for grouping IP addresses was removed from the `clusterNetwork` configuration. See the xref:../microshift_configuring/microshift-using-config-tools.adoc#microshift-yaml-default_microshift-using-config-tools[Using configuration tools] documentation for an example.
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-14-release-notes.adoc:134]: @abhatt-rh Yea, that was updated in the cross-linked PR. It hadn't synced yet on the CP.

**Pull request merged**